### PR TITLE
added the file and script to create and start the vm

### PR DESCRIPTION
**Pull request by MonetCarriere**
Instead of having the script to create and start the vm be in the setup file I think it should be in it's own separate file.

### CODE REVIEW

File: create-vm.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+if (multipass info awesome-vm)
+ then
+  echo "The VM already exists"
+ else
+  echo "✨ Creating an awesome vm for you. One moment... ✨"
+  multipass launch --name awesome-vm
+fi
+
+
+if (multipass info awesome-vm)
```
Comment (stephenlauck) [inline create-vm.sh]: You don't need a conditional here and I would move the shell command outside of the script to the README.

**Review by stephenlauck — CHANGES_REQUESTED** [review]

**Review by jlotto8 — APPROVED** [review]
Excellent

**Review by DevinBeu — APPROVED** [review]
ya this is pretty nice! it is pretty neat, ya know? it really has a vibe to it, ya know? its kindof  fire

**Review by Topher2014 — APPROVED** [review]

File: create-vm.sh (hunk 1 above)
Comment (MonetCarriere) [reply create-vm.sh]: I took your suggestion and removed the conditional

**Pull request merged**